packages/app/components/Chatbot/Chatbot.tsx

```diff
@@ -42,8 +42,9 @@ export const ChatbotComponent: React.FC = () => {
   const { cid } = router.query
   const profile = useProfile()
   const [isLoading, setIsLoading] = useState(false)
-  const [interactionId, setInteractionId] = useState<number | null>(null)
+  const [_interactionId, setInteractionId] = useState<number | null>(null)
```

Comment:
what does the underscore before do. Online it says is making var private? Just curious

Reply:
linter will complain that the value is unused, adding `_` marks it as "we are aware that it's not being used"